x-pack/test/security_solution_cypress/cypress/screens/alerts_detection_rules.ts

```diff
@@ -269,8 +269,8 @@ export const TIMELINE_TEMPLATE_VALUE = '[data-test-subj="timelineTemplatePropert
 export const INTERVAL_TITLE = '[data-test-subj="intervalPropertyTitle"]';
 export const INTERVAL_VALUE = '[data-test-subj="intervalPropertyValue"]';
 
-export const FROM_TITLE = '[data-test-subj="fromPropertyTitle"]';
-export const FROM_VALUE = '[data-test-subj^="fromPropertyValue"]';
+export const LOOK_BACK_TITLE = '[data-test-subj="lookBackPropertyTitle"]';
+export const LOOK_BACK_VALUE = '[data-test-subj^="lookBackPropertyValue"]';
```

Comment:
Same here - why did it change?

Reply:
The reason is in showing two types of rule schedule 1) simple rule schedule and 2) full rule schedule. The first one shows `lookback` but the latter shows `from` and `to`. Due to most probably legacy reasons `from` was used in the `data-test-subj` for `lookback` field. It was changes in this PR since it touched rule schedule related components.